src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,122 @@
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
+export class TodoList extends React.Component {
+  state = {
+    query: '',
+    selectedOption: '',
+  }
+
+  render() {
+    const { todos, userId, onSelection, todoId } = this.props;
+    const { query, selectedOption } = this.state;
+
+    let filteredTodos = todos;
+
+    if (query) {
+      const lowerCaseQuery = query.toLowerCase();
+
+      filteredTodos = todos
+        .filter(todo => (
+          todo.title.toLowerCase().includes(lowerCaseQuery)
+        ));
+    }
+
+    if (selectedOption === 'active') {
+      filteredTodos = todos
+        .filter(todo => (
+          todo.completed === false
+        ));
+    }
+
+    if (selectedOption === 'completed') {
+      filteredTodos = todos
+        .filter(todo => (
+          todo.completed === true
+        ));
+    }
+
+    return (
+      <div className="TodoList">
+        <h2>Todos:</h2>
+
+        <form className="TodoList__form">
+          <input
+            type="text"
+            id="search-query"
+            className="TodoList__input"
+            placeholder="Type search word"
+            value={query}
+            onChange={event => (
+              this.setState({ query: event.target.value })
+            )}
+          />
+
+          <select
+            name="select"
+            value={selectedOption}
+            className="TodoList__select"
+            onChange={event => (
+              this.setState({ selectedOption: event.target.value })
+            )}
           >
-            User&nbsp;#2
-          </button>
-        </li>
-      </ul>
-    </div>
-  </div>
-);
+            <option value="all">
+              Show all
+            </option>
+
+            <option value="active">
+              Show active
+            </option>
+
+            <option value="completed">
+              Show completed
+            </option>
+          </select>
+        </form>
+
+        <div className="TodoList__list-container">
+          <ul className="TodoList__list">
+            {filteredTodos.map(todo => (
+              <li
+                key={todo.id}
+                className={classNames('TodoList__item', {
+                  'TodoList__item--unchecked': todo.completed === false,
+                  'TodoList__item--checked': todo.completed === true,
```

Comment:
you could just do it without making a comparison
```suggestion
                  'TodoList__item--unchecked': !todo.completed,
                  'TodoList__item--checked': todo.completed,
```